src/linter/valid_examples.cc

```diff
@@ -20,7 +21,7 @@ ValidExamples::ValidExamples(Compiler compiler)
           SchemaTransformRule{"blaze/valid_examples",
                               "Only include instances in the `examples` array "
                               "that validate against the schema"},
-      compiler_{std::move(compiler)} {};
+      compiler_{std::move(compiler)}, invalid_indices_{} {};
```

Comment:
This shouldn't be needed. Containers have a default constructor that does nothing already